Assets/Scripts/PickupSystem.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickupSystem : MonoBehaviour

```

Comment:
```suggestion
public sealed class PickupSystem : MonoBehaviour
```
Dit is beetje groot mooie. Nu kan je niet ervan overerfen.